plugins/modules/system/sudoers.py

```diff
@@ -134,6 +139,8 @@ def write(self):
         with open(self.file, 'w') as f:
             f.write(self.content())
 
+        os.chmod(self.file, self.FILE_MODE)
```

Comment:
This is a bugfix and should be in a separate PR (because we need to backport it). The remainder of the PR is a feature (that must not be backported to older stable branches).

Reply:
Sure, can do that.

When I first started writing it, they were somewhat coupled as the permissions would fail the check, but since then I'm using visudo with stdin to check the file (so we don't have write test files), technically that's not an issue any more.
I suppose it could write to /tmp to not worry about the file too much afterwards.

It'll depend on whether visudo has always had the `-f -` option - as per your other comment. I'll see if I can find out.

Reply:
I have pulled out the file permissions changes from this PR and added them to a complimentary PR #4814.